out-lua/Makefile

```diff
@@ -7,7 +8,15 @@ clean:
 	rm *.so *.bc lua.shared
 
 lua.shared: _main.bc
-	$(CC) $< -L. $(patsubst %.bc,-l%,$(filter-out _main.bc,$(wildcard *.bc))) -L. -ldl -lm -g -Wl,-rpath,$(shell pwd) -o lua.shared
+	$(CC) $(CFLAGS) $< -L. -L/root/lua $(patsubst %.bc,-l%,$(filter-out _main.bc,$(wildcard *.bc))) -L. -L/root/lua -ldl -lm -g -Wl,-rpath,/root/lua -o lua.shared
 	
 lib%.so: %.bc
-	$(CC) -shared -fuse-ld=lld -fPIC $< -o lib$(basename $<).so
+	$(CC) $(CFLAGS) -shared -fPIC $< -o lib$(basename $<).so
+
+copy-lua:
+	ssh -p 10021 root@localhost "mkdir -p /root/lua"
```

Comment:
Shouldn't this be `SSHPORT` rather than 10021?

Reply:
Of course.

Reply:
Fix in bd290fb.